mne/time_frequency/tests/test_multitaper.py

```diff
@@ -33,6 +33,9 @@ def test_dpss_windows():
     assert_array_almost_equal(dpss, dpss_ni)
     assert_array_almost_equal(eigs, eigs_ni)
 
+    dpss, _ = dpss_windows(245411, 4, 8, False, 1000)
+    assert_equal(dpss.shape[-1], 245411)
```

Comment:
~1 sec. But we could get rid of it if you want. I'm + 0.5 on removing it but I was afraid of no-test complainers :)